chromatic_fitting/diagnostics/checks.py

```diff
@@ -19,3 +22,19 @@ def check_model(mod):
 def check_results(result):
     # include some result check
     return
+
+
+def check_initial_guess(model):
```

Comment:
added a function to check the initial test_point for the optimization stage - here we can maybe flag issues with unconstrained priors